src/frontend/apps/web/pages/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
+
+import { DailyViews } from "ui";
+
+const queryClient = new QueryClient();
```

Comment:
```suggestion
const queryClient = new QueryClient({
  defaultOptions: {
  queries: {
    refetchOnWindowFocus: false, // default: true
  },
  }
});
```
The default behavior of the QueryClient is to re-fetch the query on each change of focus, is this the expected behavior?

Reply:
I was also surprised by this behavior. You are right: I have changed the defaults.